bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1541,8 +1541,26 @@ use_crop=".true.">lnd/clm2/surfdata_map/ctsm5.1.dev052/landuse.timeseries_mpasa1
 
 <!-- SNICAR (SNow, ICe, and Aerosol Radiative model) datasets -->
 <!--  ***********  Resolution independent:   ***********  -->
-<fsnowoptics >lnd/clm2/snicardata/snicar_optics_5bnd_c090915.nc</fsnowoptics>
-<fsnowaging  >lnd/clm2/snicardata/snicar_drdt_bst_fit_60_c070416.nc</fsnowaging>
+<fsnowaging>lnd/clm2/snicardata/snicar_drdt_bst_fit_60_c070416.nc</fsnowaging>
+
+<snicar_numrad_snw>5</snicar_numrad_snw>
+
+<!-- TODO slevis: IF I UNCOMMENT, IT PICKS UP THE 480 FILE BY DEFAULT. WHY?
+<fsnowoptics snicar_numrad_snw=480>lnd/clm2/snicardata/snicar_optics_480bnd_c012422.nc</fsnowoptics>
+-->
+<fsnowoptics snicar_numrad_snw=5  >lnd/clm2/snicardata/snicar_optics_5bnd_c013122.nc</fsnowoptics>
```

Comment:
Things work right now, but see above TODO comment.

Reply:
@slevis-lmwg my only suggestion is to put double quotes around the setting of the number for snicar_numrad_snw. I'm pretty sure that's needed for valid XML. You might try running the file in xmllint to check for syntax issues in the file.

Reply:
Thank you @ekluzek, that was it!

Also, I tried:
`xmllint --debug namelist_defaults_ctsm.xml --output xmllint.out`
...and I see how it points out the error, while after I fix it, xmllint does not complain anymore.